lang/fortran/README.md

```diff
@@ -0,0 +1,3 @@
+Interface only for Ubuntu, otherwise you need to get the lgfortran.a file. To run the app first you need to install
```

Comment:
Please use one sentence per line and one line per sentence.
It's easier to review, and GitHub parses them the same both ways.